README.md

```diff
@@ -1,44 +1,97 @@
 
-Welcome to the next-generation repo for the ml5.js project! Due to the complexities inherent in updating all the dependencies in [the original repo](https://github.com/ml5js/ml5-library), we're exploring rebuilding from the ground up. This fresh repository allows us to migrate over classes and functions one at a time. This is very much a work in progress and is currently in its early, experimental stage. We appreciate your patience as we work towards refining this next evolution of the ml5.js project. Stay tuned!
+![ml5](https://user-images.githubusercontent.com/10605821/41332516-2ee26714-6eac-11e8-83e4-a40b8761e764.png)
 
-## Setup
+## Friendly machine learning for the web!
 
-This build uses node version 18.15.0.
-We use [Yarn](https://yarnpkg.com/) instead of npm to help us better manage dependencies from TensorFlow.js. Yarn is a newer package manager that is very similar to NPM. [Here is a cheat sheet](https://www.digitalocean.com/community/tutorials/nodejs-npm-yarn-cheatsheet) for npm vs Yarn commands.
+**Read our [ml5.js Code of Conduct and software licence](https://github.com/ml5js/Code-of-Conduct) here!**
 
-[Install nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and run the following commands：
+**_This project is currently in development._**
 
-```
-nvm install 18.15
-nvm use 18
-npm install -g yarn
-```
+ml5.js aims to make machine learning approachable for a broad audience of artists, creative coders, and students. The library provides access to machine learning algorithms and models in the browser, building on top of [TensorFlow.js](https://js.tensorflow.org/).
 
-To start the development server, run the following commands:
+The library is supported by code examples, tutorials, and sample data sets with an emphasis on ethical computing. Bias in data, stereotypical harms, and responsible crowdsourcing are part of the documentation around data collection and usage.
 
-```
-yarn
-yarn start
+ml5.js is heavily inspired by [Processing](https://processing.org/) and [p5.js](https://p5js.org/).
+
+Please read our [Code of Conduct](https://github.com/ml5js/Code-of-Conduct), which establishes our commitment to make ml5.js a friendly and welcoming environment.
+
+## Usage
+
+Before getting started with ml5.js, review our [Code of Conduct](https://github.com/ml5js/Code-of-Conduct). There are several ways you can use the ml5.js library:
+
+<p id="latest-version">
+
+- You can use the latest version (0.12.2) by adding it to the head section of your HTML document:
+
+**v0.12.2**
+
+    <script src="https://unpkg.com/ml5@0.12.2/dist/ml5.min.js" type="text/javascript"></script>
+
+</p data-id="latest-version">
```

Comment:
We used to have a script that updates the latest version info automatically. I think it would make sense to bring the script back when we look into CI options. For now, we should change it manually to `1.0.0` when making our next release.

Reply:
Agreed, however, should use change this to the `0.20` alpha version now at least temporarily?